src/services/rate-limit/index.ts

```diff
@@ -33,5 +33,14 @@ export const RedisRateLimitService = ({
     }
   }
 
-  return { consume, reset }
+  const reward = async (key) => {
```

Comment:
although is not necessary, we should try to keep the same signature defined in domain layer (readability)

Reply:
is that the fix you're looking for? https://github.com/GaloyMoney/galoy/pull/730/commits/5c6ffbf8df01bf50483bedf4b802b2d07a5fdb9c or is it something else?

Reply:
what I mean is: if you have this in domain
```js
  consume(key: string): Promise<true | RateLimitServiceError>
  reset(key: string): Promise<true | RateLimitServiceError>
  reward(key: string): Promise<true | RateLimitServiceError>
```

then the service implementation should be something like: 
```js
  const consume = async (key: string): Promise<true | RateLimitServiceError> => {...}
  const reset = async (key: string): Promise<true | RateLimitServiceError> => {...}
  const reward = async (key: string): Promise<true | RateLimitServiceError> => {...}
```

Reply:
I don't get it. this is what the code is currently doing?

![image](https://user-images.githubusercontent.com/11156370/143719294-f23a390f-fe24-47f6-a707-7bd71a245811.png)

are you suggesting it's needed to have explicit declaration of type versus implicit?

Reply:
assuming your comment is related to this, typescript is inferring the type automatically so I'm not sure why we would want to specify it manually